src/config/dynamic-theme-fixes.config

```diff
@@ -12725,6 +12709,26 @@ body {
 
 ================================
 
+pronote.toutatice.fr
+
+INVERT
+.ObjetGrille
+.Cours
+.Insecable
+.Calendrier_Jour_Selection
+
+CSS
+body,
+.ie-chips.tag-style {
+	border-radius: .1rem;
+	padding-right: 0;
+	position: relative;
```

Comment:
```suggestion
```

Don't think these are needed.

Reply:
You're right, i will also delete the background-color as it is useless, i just tested